docs/oas/business.yaml

```diff
@@ -2773,13 +2876,103 @@ paths:
                   value:
                     errorMessage: API backend third party service error.
                     rootCause: message:You are not authorized to submit a filing for TgtPxoS4FF.
+                incorporation-unauthorized-response:
```

Comment:
Consider following the naming convention  
![image](https://github.com/user-attachments/assets/2cd5ffad-2ecd-46f3-add3-9dfcf508f422)

Reply:
ideally, it will be `incorporation-failed-unauthorized-response`, same goes to the other ones. But I don't mind have a final refactor after all of these are updated